Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/Result.cs

```diff
@@ -1,140 +1,140 @@
-        /// }
-        /// </code>
-        /// </example>
-        public IDictionary<string, string> OperationsLog { get; set; }
+    /// <summary>
+    /// Operations logs for the transfer.
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// {
+    ///     "2022-05-30 12:27:35.00Z": "FILE LIST C:\\test\\test.txt"
+    ///     "2022-06-01 11:01:50.40Z": "RenameSourceFileBeforeTransfer: Renaming source file test.txt to temporary file name frends_637896781104694806az33q4kf.8CO before transfer"
+    /// }
+    /// </code>
+    /// </example>
+    public IDictionary<string, string> OperationsLog { get; set; }
 
-        internal Result(Dictionary<string, IList<string>> transferErrors)
-        {
-            TransferErrors = transferErrors;
-        }
+    internal Result(Dictionary<string, IList<string>> transferErrors)
+    {
+        TransferErrors = transferErrors;
+    }
 
-        internal Result(FileTransferResult result)
-        {
-            ActionSkipped = result.ActionSkipped;
-            Success = result.Success;
-            UserResultMessage = result.UserResultMessage;
-            SuccessfulTransferCount = result.SuccessfulTransferCount;
-            FailedTransferCount = result.FailedTransferCount;
-            TransferredFileNames = result.TransferredFileNames;
-            TransferErrors = result.TransferErrors;
-            TransferredFilePaths = result.TransferredFilePaths;
-            OperationsLog = result.OperationsLog;
-        }
+    internal Result(FileTransferResult result)
+    {
+        ActionSkipped = result.ActionSkipped;
+        Success = result.Success;
+        UserResultMessage = result.UserResultMessage;
+        SuccessfulTransferCount = result.SuccessfulTransferCount;
+        FailedTransferCount = result.FailedTransferCount;
+        TransferredFileNames = result.TransferredFileNames;
+        TransferErrors = result.TransferErrors;
+        TransferredFilePaths = result.TransferredFilePaths;
+        OperationsLog = result.OperationsLog;
+    }
 
-        internal Result(
-            bool actionSkipped, 
-            bool success, 
-            string userResultMessage, 
-            int successfulTransferCount, 
-            int failedTransferCount, 
-            IEnumerable<string> transferredFileNames, 
-            Dictionary<string, IList<string>> transferErrors, 
-            IEnumerable<string> transferredFilePaths, 
-            IDictionary<string, string> operationsLog)
-        {
-            ActionSkipped = actionSkipped;
-            Success = success;
-            UserResultMessage = userResultMessage;
-            SuccessfulTransferCount = successfulTransferCount;
-            FailedTransferCount = failedTransferCount;
-            TransferredFileNames = transferredFileNames;
-            TransferErrors = transferErrors;
-            TransferredFilePaths = transferredFilePaths;
-            OperationsLog = operationsLog;
-        }
+    internal Result(
```

Comment:
You are not using this constructor, so it can be removed.

Reply:
Removed.